src/lib/handlebars-converter/handlebars-helpers.spec.js

```diff
@@ -243,7 +243,7 @@ describe('Handlebars helpers', function () {
         });
     });
 
-    it('gzip returns an expected value', function() {
+    xit('gzip returns an expected value', function() {
```

Comment:
Should we add a comment for why this is skipped?

Reply:
sure